tsconfig.json

```diff
@@ -29,15 +29,16 @@
       "providers": ["providers/src/*"],
       "drops": ["drops/src"],
       "utils": ["utils/src"],
-      "poaps": ["poaps/src"]
-    }
+      "poaps": ["poaps/src"],
+    },
```

Comment:
I think JSON doesn't accept `,` in the last item. Does it work for you?

Reply:
I would think the same, but all of our tsconfigs have them. It's Prettier's doing.